pkg/generic/thrift/base.go

```diff
@@ -19,7 +19,8 @@ package thrift
 import (
 	"fmt"
 
-	"github.com/cloudwego/kitex/pkg/protocol/bthrift"
+	bthrift "github.com/cloudwego/gopkg/protocol/thrift"
```

Comment:
The names like followings are easier to understand? (need to refactor though)
```
"github.com/cloudwego/gopkg/protocol/thrift"
athrift "github.com/cloudwego/kitex/pkg/protocol/bthrift/apache"
```

Reply:
the base implementation will be removed soon. I will provide a new one. so should be OK, don't want to touch it actually...